ui/src/components/Issuer/Issuer.tsx

```diff
@@ -36,38 +34,46 @@ const Issuer: React.FC<Props> = ({ onLogout }) => {
     const ledger = useLedger();
 
     const registeredIssuer = useStreamQuery(RegisteredIssuer);
-    const issuerModel = useStreamQuery(IssuerModel);
-
     const allCustodianRelationships = useStreamQuery(CustodianRelationship).contracts.map(makeContractInfo);
 
     const [ profile, setProfile ] = useState<Profile>({
         'name': createField('', 'Name', 'Your full legal name', 'text'),
+        'location': createField('', 'Location', 'Your current location', 'text'),
         'title': createField('', 'Title', 'Your professional title', 'text'),
         'issuerID': createField('', 'Issuer ID', 'Your Issuer ID', 'text'),
-        'ssn': createField('', 'Social Security Number (private)', 'Your social security number', 'password')
     });
 
     useEffect(() => {
         if (registeredIssuer.contracts[0]) {
             const riData = registeredIssuer.contracts[0].payload;
-            const issuerContract = issuerModel.contracts[0];
-            const ssn = issuerContract ? issuerContract.payload.ssn : 'Private';
             setProfile({
                 name: { ...profile.name, value: riData.name },
+                location: { ...profile.location, value: riData.location },
                 title: { ...profile.title, value: riData.title },
-                issuerID: { ...profile.issuerID, value: riData.issuerID },
-                ssn: { ...profile.ssn, value: ssn },
+                issuerID: { ...profile.issuerID, value: riData.issuerID }
```

Comment:
i dont remember but is the convention in the marketplace camel case with `somethingID` or `somethignId` (like dabl)

Reply:
the daml used `somethingID`, so I kept the convention here

Reply:
Bit confusing since `somethingId` means `ContractID` by convention, but "Issuer ID" is one of the fields in a list of potential role information from a document @dimitri-da shared with me (that seems to have gotten deleted by Slack's 2 week policy).